zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterSettingManager.java

```diff
@@ -1038,8 +1038,10 @@ private boolean inlineRemove(String id, boolean initiator) throws IOException {
       removed = true;
     }
 
-    File localRepoDir = new File(conf.getInterpreterLocalRepoPath() + "/" + id);
-    FileUtils.deleteDirectory(localRepoDir);
+    File localRepoPath = new File(conf.getInterpreterLocalRepoPath());
+    if (Arrays.stream(localRepoPath.list()).anyMatch(id::equals)) {
```

Comment:
It is not necessary to do a comparison again, because Zeppelin doesn't allow interpreter setting id to contain dot

Reply:
ping @iiiusky

Reply:
I'm sorry, I've been busy these two days, can you give me some time to deal with my own affairs first? Submit the code by next week at the latest

Reply:
Sure, take it easy